assets/css/src/widgets.css

```diff
@@ -37,7 +37,7 @@
 
 /* Archive and Categories widgets */
 .widget_archive li {
-	color: #7d7d7d;
+	color: #808080;
```

Comment:
I think having this is useless and weird. Why would we want to show this _very_ specific list in a different color? I think this rule should be removed entirely.

Reply:
> I think this rule should be removed entirely.

Yes. This widget displays a list of links, this color is never used (overriden by the link color)